### ISSUE TITLE
Additional Inspections on core config YML files

### ISSUE THREAD

**Issue by Narimm** [DemonWav/MinecraftDevIntelliJ#366]
Please include the following information in all issues:

* Minecraft Development for IntelliJ plugin version Version: 2017.3-1.1.1
* IntelliJ version 2017.3.4
* Operating System (this has come up enough times to be important) Win10
* Target Platform: Bukkit
* Feature - additial inspections.

Feature Request(Trival)

This probably seems trivial but I would request code completion options and possibly some code check sanity checks for the various config files ie (plugin.yml for Bukkit and bungee.yml for spigot).

Checks to ensure the presence of a 'mainClass' (and that it actually corresponds to a class), name, version at a minimum but other code completion options would be nice...

**Comment by DenWav** [comment]
I agree, this should be supported. This was a major target of this plugin until Mixins, MCP, and Forge became the primary targets of this plugin.

**Comment by MarcinWieczorek** [comment]
Also, a newly created class extending JavaPlugin should suggest to be added to `plugin.yml`.
